src/main/java/org/fcrepo/spec/testsuite/authz/WebACModes.java

```diff
@@ -160,9 +311,17 @@ public void writeDisallowed(final String uri) {
     @Parameters({"param1"})
     public void appendAllowedPOST(final String uri) {
         final TestInfo info = setupTest("5.0-N", "appendAllowedPOST",
-                "acl:Append gives a more limited ability to write to a resource -- Append-Only. " +
-                        "This generally includes the HTTP verb POST.",
-                "https://fedora.info/2018/06/25/spec/#resource-authorization", ps);
+                                        "acl:Append gives a more limited ability to write to a resource -- " +
+                                        "Append-Only. " +
+                                        "This generally includes the HTTP verb POST.",
+                                        "https://fedora.info/2018/06/25/spec/#resource-authorization", ps);
+        //create a resource
+        final String resourceUri = createResource(uri, info.getId());
+        final String aclUri = createAclForResource(resourceUri, "user-read-append.ttl", this.username);
+        //perform POST  to child resource as non-admin
+        doPost(resourceUri, new Headers(new Header("Slug", "child")),
```

Comment:
As @bseeger [notes](https://github.com/fcrepo/Fedora-API-Test-Suite/pull/150#discussion_r212671999), we may want to begin getting away from the use of `slugs`s.